tests/acceptance/helpers/webdavHelper.js

```diff
@@ -5,8 +5,9 @@ const convert = require('xml-js')
 const _ = require('lodash/object')
 const { normalize, join, filename } = require('../helpers/path')
 const occHelper = require('../helpers/occHelper')
-let timeOfLastUploadOperation = Date.now()
-let timeOfLastDeleteOperation = Date.now()
+
+const uploadTimeStamps = {}
+const deleteTimestamp = {}
```

Comment:
```suggestion
const deleteTimeStamps = {}
```
to be consistent with `uploadTimeStamps`